src/js/components/Notification.js

```diff
@@ -3,16 +3,34 @@
 import React, { Component, PropTypes } from 'react';
 import classnames from 'classnames';
 import { FormattedDate } from 'react-intl';
+import Intl from '../utils/Intl';
 import Box from './Box';
+import Animate from './Animate';
 import Meter from './Meter';
+import Button from './Button';
 import StatusIcon from './icons/Status';
+import CloseIcon from './icons/base/Close';
 import Props from '../utils/Props';
 import CSSClassnames from '../utils/CSSClassnames';
 
 const CLASS_ROOT = CSSClassnames.NOTIFICATION;
 const BACKGROUND_COLOR_INDEX = CSSClassnames.BACKGROUND_COLOR_INDEX;
 
 export default class Notification extends Component {
+
+  constructor (props, context) {
+    super(props, context);
+
+    this._onClickClose = this._onClickClose.bind(this);
+    this.state = {
+      isHidden: false
+    };
+  }
+
+  _onClickClose () {
```

Comment:
I believe it is better for the caller to close it. All we have to do is to provide a callback function that the caller will be using to remove the Notification. We do the same with Layer and Tip.